gdcdictionary/schemas/sample.yaml

```diff
@@ -485,7 +487,6 @@ properties:
       - Wedge Resection
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   oct_embedded:
     term:
       $ref: "_terms.yaml#/oct_embedded"
```

Comment:
Can't seem to comment on the line - but remove `pathology_report_uuid`? As the comment says, should be handled.

Reply:
Removed `pathology_report_uuid`